src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/EnumsShouldHavePluralNames.cs

```diff
@@ -108,7 +108,7 @@ public override void Initialize(AnalysisContext analysisContext)
         private static void AnalyzeSymbol(SymbolAnalysisContext context, INamedTypeSymbol flagsAttribute)
         {
             var symbol = (INamedTypeSymbol)context.Symbol;
-            if (symbol.TypeKind != TypeKind.Enum)
+            if (symbol.TypeKind != TypeKind.Enum || !symbol.IsExternallyVisible())
```

Comment:
what happens for protected? is this right definition for IsExternallyVisible?

http://index/?query=IsExternallyVisible&rightProject=Analyzer.Utilities&file=Extensions%5CISymbolExtensions.cs&rightSymbol=rccoswinkudx

Reply:
This is the definition for this method: https://github.com/dotnet/roslyn-analyzers/blob/master/src/Utilities/Extensions/ISymbolExtensions.cs#L91

Reply:
I see, so it treat protected as public.